xbmc/platform/linux/WebOSTVPlatformConfig.cpp

```diff
@@ -13,7 +13,8 @@
 #include "utils/Variant.h"
 #include "utils/log.h"
 
-#include <webos-helpers/libhelpers.h>
+HContext WebOSTVPlatformConfig::m_requestContextARC{};
```

Comment:
For a proper cleanup we need to call `HUnregisterServiceCallback` when finished. The easiest way would be to wrap this in a `std::unique_ptr` with a custom destructor that calls the function